src/js/api/urls.js

```diff
@@ -146,3 +146,4 @@ export const PRODUCT_SUPPLIER_ATTRIBUTE_BATCH = `${PRODUCT_SUPPLIER_ATTRIBUTE_AP
 
 export const PICKLIST_API = `${API}/picklists`;
 export const PICKLIST_CLEAR = id => `${PICKLIST_API}/${id}/items`;
+export const PICKLIST_REVERT_PICK = (id, itemId) => `${PICKLIST_CLEAR(id)}/${itemId}`;
```

Comment:
We should be more obvious here to avoid confusion / regression later. If there's a baseUrl shared by multiple endpoints, let's define the base URL. Also
```
export const PICKLIST_API = `${API}/picklists`;
export const PICKLIST_ITEM_API = `${API}/picklists/${id}/items`;
export const PICKLIST_ITEM_CLEAR = id => `${PICKLIST_ITEMS_API}`;
export const PICKLIST_ITEM_REVERT = (id, itemId) => `${PICKLIST_ITEMS_API(id)}/${itemId}`;
```

Reply:
:+1: this could be improved